modules/data-loading/partials/kafka/kafka-example-loading-job.adoc

```diff
@@ -2,7 +2,7 @@
 
 The following is an example loading job from and external Kafka cluster.
 
-[source,php,linenums]
+[source,sql,linenums]
```

Comment:
I believe this is not the only place using `php`, could you check other code blocks?